app/models/SuppliersModel.php

```diff
@@ -0,0 +1,50 @@
+<?php
+class SuppliersModel{
+
+    private $table = 'suppliers';
+    private $db;
+
+    public function __construct()
+    {
+        $this->db = new Database;
+    }
+
+    public function getAllSuppliers()
+    {
+        $this->db->query('SELECT * FROM '.$this->table);
+        return $this->db->resultSet();
+    }
+
+    public function getSuppliersById($id)
+    {
+        $this->db->query('SELECT * FROM '.$this->table.' WHERE id=:id');
+        $this->db->bind('id', $id);
+
+        return $this->db->single();
+    }
+
+   public function storeSuppliersData($data)
+    {    
+        $query = "INSERT INTO suppliers
```

Comment:
try {
            # your code should be here
            
        } catch (\PDOException $e) {
            $e->getMessage();
        }